pitop/miniscreen/oled/core/controls.py

```diff
@@ -6,6 +6,9 @@
 from luma.core.interface.serial import spi
 from luma.oled.device import sh1106
 
+from pitop.system.daemon import get_oled_spi_port
```

Comment:
Move the request client logic into this file. You may need to turn this file into a class of sorts to facilitate this.
This is something that will need to happen sooner or later anyway, so probably makes sense now.

Reply:
Use battery as a guide